packages/FormElement/src/components/Content/Content.js

```diff
@@ -26,14 +24,14 @@ function Content(props) {
   const ariaDescribedByIdsString = ariaDescribedByIdsArray.join(" ");
 
   const a11yProps = {
-    refLabel,
+    "aria-describedby": ariaDescribedByIdsString.length ? ariaDescribedByIdsString : null,
+    "aria-required": isRequired || null,
+    "aria-disabled": isDisabled || null,
+    disabled: isDisabled || null,
```

Comment:
It seems to work like this for `Input`, `Select` and `Textarea`.  If you add `isDisabled` to the `<Input>` _too_ you get a _very_ faint looking input, since the whole `<FormElement>` gets `opacity: 0.5` when it has `isDisabled`.  But that's 🤷‍♂️ 

I guess you might get weirdness if you set `isDisabled` on `FormElement` and `isDisabled={false}` on the `<Input>` but why would you ever do that?

If you _only_ set `isDisabled` on the `<Input>` that's fine too, the rest of the `<FormElement>` pieces just won't be dimmed.